diff -up eid-mw-5.1.16-v5.1.16/configure.ac.omv~ eid-mw-5.1.16-v5.1.16/configure.ac
--- eid-mw-5.1.16-v5.1.16/configure.ac.omv~	2024-07-06 02:42:38.584766112 +0200
+++ eid-mw-5.1.16-v5.1.16/configure.ac	2024-07-06 02:43:05.057062488 +0200
@@ -204,7 +204,7 @@ AM_CONDITIONAL([JPEG], [test x$HAVE_JPEG
 AC_DEFINE_UNQUOTED([HAVE_JPEGLIB], $HAVE_JPEGLIB, [Define to 1 if you have libjpeg installed])
 
 PKG_CHECK_MODULES(SSL, "openssl", [HAVE_OPENSSL=1])
-PKG_CHECK_VAR(SSL_PREFIX, "openssl", "exec_prefix",, AC_MSG_ERROR([Could not find exec_prefix for openssl]))
+PKG_CHECK_VAR(SSL_PREFIX, "openssl", "exec_prefix",, [SSL_PREFIX="/usr"])
 AC_DEFINE_UNQUOTED([HAVE_OPENSSL], $HAVE_OPENSSL, [Define to 1 if you have OpenSSL installed])
 AC_SUBST(SSL_PREFIX)
 
